{"name": "test_phones_on_contact_view_page", "status": "broken", "statusDetails": {"message": "AttributeError: module 'pytest' has no attribute 'allure'", "trace": "app = <fixture.application.Application object at 0x03AEFC58>\n\n    def test_phones_on_contact_view_page(app):\n>       with pytest.allure.step('Get contacts from viewpage'):\nE       AttributeError: module 'pytest' has no attribute 'allure'\n\ntest\\test_phones.py:15: AttributeError"}, "start": 1603202626967, "stop": 1603202626967, "uuid": "44418796-3309-442d-b02c-b38b3bdb7420", "historyId": "20082a8709fda6e17eb1a099fbf885ed", "testCaseId": "29827f509b35331c3f296325c05d9595", "fullName": "test.test_phones#test_phones_on_contact_view_page", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_phones"}, {"name": "host", "value": "WIN-BP2DE13KB1K"}, {"name": "thread", "value": "11728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_phones"}]}